luci_grpc_interface/sensors_grpc/sensors.proto

```diff
@@ -339,4 +342,16 @@ message ChairProfile {
 
 message SpeedSetting {
   uint32 speed_setting = 1;
+}
+
+message HmiStreamData {
```

Comment:
Was the additional info about whether this is from the LUCI button or the alt port intentionally left out? I feel like that could be useful to include unless there's a reason not to

Reply:
Discussed this and it was left out to avoid extra confusion and it was deemed unnecessary